webf/lib/src/dom/document.dart

```diff
@@ -150,6 +150,21 @@ class Document extends Node {
     }
   }
 
+  int _domContentLoadedEventDelayCount = 0;
+  bool get isDelayingDOMContentLoadedEvent => _domContentLoadedEventDelayCount > 0;
+  void incrementDOMContentLoadedEventDelayCount() {
```

Comment:
Where is this function called?

Reply:
https://github.com/openwebf/webf/blob/main/webf/lib/src/html/script.dart#L63